lib/utils/translateResults.js

```diff
@@ -19,37 +20,29 @@ const transformMarkdown = md => md
 	}); // links, previews -> text + url
 
 /**
- * Applies tranformMarkdown to all strings in input
- * @param {*} input - object/array/string/any
- * @returns {*} input with transformed strings
+ * Composes output text from result object with title and description.
+ * @param {{title: [{type: string, value: string}], description: [{type: string, value: string}]}} result
+ * @returns {string} - composed text
  */
-const transformAllMarkdowns = input => {
-	if (typeof input === 'string') {
-		return transformMarkdown(input);
-	}
-	if (typeof input === 'object' && input !== null) {
-		return map(transformAllMarkdowns, input); // array or object
+
+const titleDescToText = (result) => {
+	const titleResult = text.toText(result.title, {});
+
+	if (result.description) {
+		const descriptionResult = text.toText(result.description, {});
+
+		return `${titleResult}\n${descriptionResult}`;
 	}
 
-	return input;
+	return `${titleResult}`;
```

Comment:
Minor thing, but you don't really need string interpolation here. Just `return titleResult`.